contracts/scripts/runGanache.sh

```diff
@@ -21,3 +21,6 @@ npx etherlime ganache --mnemonic "candy maple cake sugar pudding cream honey ric
 #7: 0x0f4f2ac550a1b4e2280d04c21cea7ebd822934b5
 #8: 0x6330a553fc93768f612722bb8c2ec78ac90b3bbc
 #9: 0x5aeda56215b167893e80b4fe645ba6d5bab767de
+
+# Private keys
```

Comment:
Is this accidentally checked in?

Reply:
Fortunately not!

This mnemonic used to be the default Truffle suite mnemonic. https://www.trufflesuite.com/docs/truffle/getting-started/using-truffle-develop-and-the-console

As such these private keys were compromised a long time ago.